net.poedit.Poedit.json

```diff
@@ -147,6 +149,14 @@
                     "path": "appdata.patch"
                 }
             ]
+        },
+        {
+            "name": "poedit-fixup",
+            "buildsystem": "simple",
+            "build-commands": [
+                /* Export PoEdit URI handler under allowed name */
+                "mv /app/share/applications/net.poedit.PoeditURI.desktop /app/share/applications/net.poedit.Poedit.URI.desktop"
```

Comment:
You can do that in `post-install` of the `poedit` module.

Reply:
Done!